contribs/gmf/src/directives/mobilenav.js

```diff
@@ -286,3 +286,44 @@ gmf.mobileNavBackDirective = function() {
 };
 
 gmf.module.directive('gmfMobileNavBack', gmf.mobileNavBackDirective);
+
+
+/**
+ * A directive to be used in conjunction with {@link gmf.mobileNavDirective}.
+ * The directive can be set on a slide element of {@link gmf.mobileNavDirective}.
+ * When the element is clicked, the navigation returns to the previous slide if
+ * the slide is currently active.
+ *
+ * Example:
+ *
+ *    <nav class="nav-left" gmf-mobile-nav>
+ *      ...
+ *      <gmf-themeselector
+ *         gmf-mobile-nav-back-on-click
+ *         gmf-themeselector-currenttheme="mainCtrl.theme">
+ *      </gmf-themeselector>
+ *
```

Comment:
Sorry I just see the attributes from the theme selector, add ..., forget my comment :-)